web_archiving/readme.md

```diff
@@ -49,7 +49,7 @@ Capabilities are evaluated as "out of the box" functionality. In cases where a s
 
 * _**WUI**:_ Whether the software offers a web-based user interface.
 
-* _**Library API**:_ Does it provide an explicit libary interface? (I.e., does it provide an API so that you can use it in your own program?)
+* _**Library API**:_ Does it provide an explicit libary interface? (i.e., does it provide an API so that you can use it in your own program?)
```

Comment:
This proposed edit is actually incorrect: the parenthesized sentence is a full sentence, and therefore, should begin with a capital letter. 

Here are some references to back that up (admittedly not authoritative, but I don't think it's worth spending a lot of time on this):

- https://english.stackexchange.com/a/37204
- https://ell.stackexchange.com/questions/56861/capitalize-i-e-and-e-g

(We can debate whether it's good style to begin a sentence with "i.e." or "e.g.", and in fact, I generally avoid it in more formal writing.)